src/UDS.Net.API/Entities/D1b.cs

```diff
@@ -0,0 +1,316 @@
+        public int? FDGPETDX { get; set; }
+
+        [Comment("6b1. FDG PET - Consistent with AD")]
+        public int? FDGAD { get; set; }
+
+        [Comment("6b2. FDG PET - Consistent with FTLD")]
+        public int? FDGFTLD { get; set; }
+
+        [Comment("6b3. FDG PET - Consistent with LBD")]
+        public int? FDGLBD { get; set; }
+
+        [Comment("6b4. FDG PET - Consistent with other etiology")]
+        public int? FDGOTH { get; set; }
+
+        [Comment("6b4a. FDG PET - Consistent with other etiology (specify)")]
+        [MaxLength(60)]
+        public string? FDGOTHX { get; set; }
+
+        [Comment("6c. Dopamine Transporter (DAT) Scan - Was DAT Scan data or information used to support an etiological diagnosis?")]
+        public int? DATSCANDX { get; set; }
+
+        [Comment("6d. Other tracer-based imaging - Were other tracer-based imaging used to support an etiological diagnosis?")]
+        public int? TRACOTHDX { get; set; }
+
+        [Comment("6d1a. Other tracer-based imaging - Were other tracer-based imaging used to support an etiological diagnosis? (specify)")]
+        [MaxLength(60)]
+        public string? TRACOTHDXX { get; set; }
+
+        [Comment("6d1. Other tracer-based imaging - Consistent with AD")]
+        public int? TRACERAD { get; set; }
+
+        [Comment("6d2. Other tracer-based imaging - Consistent with FTLD")]
+        public int? TRACERFTLD { get; set; }
+
+        [Comment("6d3. Other tracer-based imaging - Consistent with LBD")]
+        public int? TRACERLBD { get; set; }
+
+        [Comment("6d4. Other tracer-based imaging - Consistent with other etiology")]
+        public int? TRACEROTH { get; set; }
+
+        [Comment("6d4a. Other tracer-based imaging - Consistent with other etiology (specify)")]
+        [MaxLength(60)]
+        public string? TRACEROTHX { get; set; }
+
+        [Comment("7a. Structural Imaging (i.e., MRI or CT) - Was structural imaging data or information used to support an etiological diagnosis?")]
+        public int? STRUCTDX { get; set; }
+
+        [Comment("7a1. Atrophy pattern conistent with AD")]
+        public int? STRUCTAD { get; set; }
+
+        [Comment("7a2. Atrophy pattern consistent with FTLD")]
+        public int? STRUCTFTLD { get; set; }
+
+        [Comment("7a3. Consistent with cerebrovascular disease (CVD)")]
+        public int? STRUCTCVD { get; set; }
+
+        [Comment("7a3a. Large vessel infarct(s)")]
+        public int? IMAGLINF { get; set; }
+
+        [Comment("7a3b. Lacunar infarct(s)")]
+        public int? IMAGLAC { get; set; }
+
+        [Comment("7a3c. Macrohemorrhage(s)")]
+        public int? IMAGMACH { get; set; }
+
+        [Comment("7a3d. Microhemorrhage(s)")]
+        public int? IMAGMICH { get; set; }
+
+        [Comment("7a3e. Moderate white-matter hyperintensity (CHS score 5-6)")]
+        public int? IMAGMWMH { get; set; }
+
+        [Comment("7a3f. Extensive white-matter hyperinensity (CHS score 7-8+)")]
```

Comment:
Looks like the documentation has a spelling error in the question. Couldn't hurt to fix this up for when we copy the text over to the web project to use as labels